basic/src/test/java/org/jboss/hal/testsuite/test/configuration/singleton/SingletonPolicyTestCase.java

```diff
@@ -139,11 +140,16 @@ public void editCacheContainer() throws Exception {
                 .navigateToTreeItem()
                 .clickLabel();
 
-        new ConfigChecker.Builder(client, SINGLETON_POLICY_ADDRESS)
-                .configFragment(page.getConfigFragment())
-                .editAndSave(ConfigChecker.InputType.TEXT, CACHE_CONTAINER, CACHE_CONTAINER_RESOURCE_EDIT)
-                .verifyFormSaved()
-                .verifyAttribute(CACHE_CONTAINER, CACHE_CONTAINER_RESOURCE_EDIT);
+        try {
+            new ConfigChecker.Builder(client, SINGLETON_POLICY_ADDRESS)
+                    .configFragment(page.getConfigFragment())
+                    .editAndSave(ConfigChecker.InputType.TEXT, CACHE_CONTAINER, CACHE_CONTAINER_RESOURCE_EDIT)
+                    .verifyFormSaved()
+                    .verifyAttribute(CACHE_CONTAINER, CACHE_CONTAINER_RESOURCE_EDIT);
+        } catch (NoSuchElementException e) {
+            //TODO remove this when issue is resolved
```

Comment:
Please remove the TODO comment. We don't want to remove regression assertion messages.